doc/guides/bluetooth/autopts/autopts-win10.rst

```diff
@@ -392,3 +407,32 @@ At the first run, when Windows asks, enable connection through firewall:
    :height: 450
    :width: 600
    :align: center
+
+Troubleshooting
+================
+
+- "When running actual hardware test mode, I have only BTP TIMEOUTs."
+
+This is a problem with connection between autopts client and board. There are many possible causes. Try:
+
+- Clean your auto-pts and zephyr repos with
+
+.. code-block::
+
+    git clean -fdx
+
+then build and flash tester elf again.
```

Comment:
This should come with a warning for those that do not realize what `git clean -fdx` will do